docs/docsite/rst/porting_guide_2.3.rst

```diff
@@ -0,0 +1,193 @@
+.. _porting_2.1_guide:
+
+*************************
+Ansible 2.3 Porting Guide
+*************************
+
+This section discusses the changes to how you use Ansible between Ansible 2.2 and Ansible 2.3.
```

Comment:
"This section discusses the behavioral changes between Ansible 2.2 and Ansible 2.3."